src/PermMissingOrzech.java

```diff
@@ -0,0 +1,19 @@
+import java.util.*;
+
+public class PermMissingOrzech {
+    int solution(int [] A) {
+        LinkedList<Integer> listA=new LinkedList<>();
```

Comment:
Nie zamieniaj na linkedList to jest niepotrzebny kod i iteracja po wszystkich elementach, zrob to na tablicy